vernac/vernacprop.ml

```diff
@@ -0,0 +1,53 @@
+(************************************************************************)
```

Comment:
shouldn't this code end up ion vernac_classifier?  Yes, it is not exactly the same output (bool v.s. something more fine grained) but semantically this is a classafication...